MapboxDirections/MBRouteLeg.swift

```diff
@@ -34,15 +38,18 @@ open class RouteLeg: NSObject, NSSecureCoding {
             if let speed = jsonAttributes["speed"] {
                 segmentSpeeds = speed as? [CLLocationSpeed] ?? []
             }
-            if let nodes = jsonAttributes["nodes"] {
-                openStreetMapNodeIdentifiers = nodes as? [Int64] ?? []
+            if let congestion = jsonAttributes["congestion"] as? [String] {
+                congestionLevels = congestion.map {
+                    CongestionLevel(description: $0)!
```

Comment:
This will crash if the Directions API returns an unrecognized congestion annotation (like “parking-lot”; “unknown” is fine). Alternatively, we could fall back to `.unknown` or throw an error. It would be inadvisable to omit the value entirely, since the developer expects the array’s length to match the number of segments in the route geometry.

The Directions API reserves the right to introduce new values without notice, which is why we have to fail gracefully when encountering an unknown maneuver type. But if I’m not mistaken, congestion values are tied to the [Mapbox Traffic v1](https://www.mapbox.com/vector-tiles/mapbox-traffic-v1/#traffic) tile set in some fashion. Renaming or introducing new values would probably require a version bump of that tile set at least, right?

/cc @freenerd